roles/inherit_kerberos/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+inherit_kerberos_config: '/etc/krb5.conf'
+inherit_kerberos_ccache: False
```

Comment:
I'd not define this as false (it's a path in reality), but check for "is defined" below?

Reply:
Is there a particular reason to prefer one of these patterns over the other?

```yaml
  when: inherit_kerberos_ccache is defined
```

vs.

```yaml
  when: inherit_kerberos_ccache | default(False)
```

I've seen both used pretty frequently and I'm wondering which is "more correct" and why.

Reply:
IMHO only `is defined` is "correct" in the case where `inherit_kerberos_ccache` is not a boolean as such.

If I am reading it correctly, the logic here is: "The user can opt-in to provide additional data (path) to be copied into the VM", which means that the variable is either undefined (the user did not opt-in) or a string/path (the user wants this path to be copied). If Ansible would be type-safe (lol), you wouldn't be able to pass a boolean here, as it's not a string. And the `default(False)` should also break.

It would be different if the variable itself would already denote a boolean (do/don't copy), where you could just set the default to `false` in the defaults file.